docs/idle-crafting.rst

```diff
@@ -0,0 +1,56 @@
+idle-crafting
+=============
+
+.. dfhack-tool::
+    :summary: Allow dwarfs to automatically satisfy their need to craft objects.
```

Comment:
```suggestion
    :summary: Allow dwarves to independently craft objects when they have the need.
```

Reply:
I think this phrasing might be better for a "gameplay" tool. As written, I'd classify it more as an "automation" tool.